### DISCUSSION TITLE
Using source code to DeDRM .drmz / .drmx files?

### DISCUSSION THREAD

**Discussion by mybig-debug** [apprenticeharper/DeDRM_tools#1695] [Q&A]
Hello! I've run into an issue where I need to remove DRM from .drmz or .drmx files, but can't find a program that handles those (Javelin3 is the normal reader, using Drumlin Security). Calibre can't take in these file types, so using the plug-in won't be possible. Is the source code suitable to run from the command line in some way to work on these file types? Or is there some other method I could try? Thanks!

**Comment by ElleKayEm** [comment]
The Python scripts or plugin, these tools do not work on that DRM.  Only Amazon, Adobe epubs or pdfs, Kobo, and B&N (sort of).

**Reply by mybig-debug** [reply]
That's what I assumed and was afraid of... I appreciate your response! Hopefully someone knows another method to work with these types
